src/NuGet.Core/NuGet.Versioning/VersionRangeFormatter.cs

```diff
@@ -11,9 +11,6 @@ namespace NuGet.Versioning
     /// </summary>
     public class VersionRangeFormatter : IFormatProvider, ICustomFormatter
     {
-        private const string ZeroN = "{0:N}";
```

Comment:
This was used to tell `VersionFormatter` to run its `AppendNormalized` method. This PR just makes that method internal and calls it directly to avoid a bunch of indirection, in addition to the temporary string allocations.